local-volume/provisioner/pkg/common/common.go

```diff
@@ -58,6 +58,8 @@ const (
 	// ProvisonerStorageClassConfig defines file name of the file which stores storage class
 	// configuration. The file name must match to the key name used in configuration map.
 	ProvisonerStorageClassConfig = "storageClassMap"
+	// ProvisionerNodeLabelsForPV List of Node labels to be copied to the PVs created by the Provisioner
```

Comment:
/List/contains a list/
/Node labels/node labels/
/Provisioner/provisioner./

I like this comment. Please copy the changes to L118.

Reply:
Fixed.